### ISSUE TITLE
CJS/ES warning when using with Vite

### ISSUE THREAD

**Issue by wardou2** [dane-harnett/cypress-cucumber-attach-screenshots-to-failed-steps#6]
I'm using this package with cypress, cypress-cucumber-preprocessor, and multiple-cucumber-html-reporter to test a SvelteKit project (which uses Vite under the hood).

The warning: 

```cypress-cucumber-attach-screenshots-to-failed-steps doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.```

Is this something that can be easily addressed here?

**Comment by dane-harnett** [comment]
Yes I could look into that, are you using any other cypress plugins other than the ones mentioned?

Would you be able to provide a minimal reproducible example that I could check some changes against?
